src/explore-education-statistics-admin/src/pages/methodology/edit-methodology/status/components/__tests__/MethodologyStatusForm.test.tsx

```diff
@@ -0,0 +1,256 @@
+import MethodologyStatusForm, {
+  FormValues,
+} from '@admin/pages/methodology/edit-methodology/status/components/MethodolodyStatusForm';
+import { Release } from '@admin/services/releaseService';
+import { render, screen, waitFor, within } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import React from 'react';
+import noop from 'lodash/noop';
+import { BasicMethodology } from 'src/services/methodologyService';
+
+describe('MethodologyStatusForm', () => {
+  const testUnpublishedReleases: Release[] = [
+    {
+      id: 'test-release-1',
+      title: 'Test Release 1',
+    } as Release,
+    {
+      id: 'test-release-2',
+      title: 'Test Release 2',
```

Comment:
For spice, we could change the order of these around so that we can make sure that the releases are ordered in alphabetical order. This re-ordering happens automatically within `FormSelect`, but we should make sure that it is indeed happening